<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInfo</class>
 <widget class="QDialog" name="AddInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>444</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="addfriend">
    <attribute name="title">
     <string>添加好友</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>16</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
     <property name="text">
      <string>帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
     <property name="text">
      <string>昵称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="friend_id">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>16</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="friend_name">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>38</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="male">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>18</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
     <property name="text">
      <string>男</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="female">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
    <widget class="QPushButton" name="friend_searchbtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt</string>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="friend_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>495</width>
       <height>360</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>昵称</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>系别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="addgroup">
    <attribute name="title">
     <string>添加群</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt</string>
     </property>
     <property name="text">
      <string>聊天室ID：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="group_id">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="group_searchbtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="group_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>495</width>
       <height>341</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>群名称</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt</string>
     </property>
     <property name="text">
      <string>聊天室名:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="group_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="creatgroup">
    <attribute name="title">
     <string>创建群</string>
    </attribute>
    <widget class="QPushButton" name="c_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>创建</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>255</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt</string>
        </property>
        <property name="text">
         <string>聊天室ID：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="c_group_id">
        <property name="styleSheet">
         <string notr="true">font: 12pt</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 12pt</string>
        </property>
        <property name="text">
         <string>聊天室名:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="c_group_name">
        <property name="styleSheet">
         <string notr="true">font: 12pt</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
